src/components/learner-credit-management/invite-modal/EnterpriseCustomerUserDatatable.jsx

```diff
@@ -0,0 +1,188 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  DataTable,
+  Stack,
+  TextFilter,
+} from '@openedx/paragon';
+import { useEnterpriseLearnersTableData, getAllEnterpriseLearners } from '../data/hooks/useEnterpriseLearnersTableData';
+import { formatTimestamp } from '../../../utils';
+import { DEFAULT_PAGE } from '../data';
+
+const getSelectedEmailsByRow = (selectedFlatRows) => {
+  const emails = [];
+  Object.keys(selectedFlatRows).forEach(key => {
+    const { original } = selectedFlatRows[key];
+    emails.push(original.user.email);
+  });
+  return emails;
+};
+
+const MemberDetailsCell = ({ row }) => (
+  <Stack gap={1}>
+    <div className="font-weight-bold">
+      {row.original.user.username}
```

Comment:
At this link https://localhost.stage.edx.org:1991/exec-ed-2u-integration-qa/admin/people-management, if I try to navigate to the 5th page, the page errors out with `Cannot read properties of null (reading 'username')`. This is probably just bad stage data, but we should handle it.

Reply:
Also, when I tried to all all members and submit, I got the error `Cannot read properties of undefined (reading 'toLowerCase')` which is probably also stage data gross-ness but should be handled.

Reply:
Yep! this is because of bad staged data. this customer has learners where the user object is `null`. 
![Screenshot 2024-11-08 at 7 08 07 AM](https://github.com/user-attachments/assets/ae629780-d621-4991-bcb8-4e3b5f84add9)